src/message/modalcomponent.js

```diff
@@ -110,11 +110,11 @@ ngeo.message.modalComponent.Controller_ = class {
   }
 
   $onInit() {
-    this.closable = this.closable === true;
+    this.closable = this.closable === false;
```

Comment:
`=== true` or `!== false` !

Reply:
`!== false` would be better, effectively
(then keep the `!` in the if below)

Reply:
@sbrunner @ger-benjamin Thanks, done.